lib/tasks/era_committees.rake

```diff
@@ -0,0 +1,35 @@
+namespace :era_committee do
+  desc 'Populate the era committees for players'
+  task populate: :environment do
+    Player.connection.execute %{
+      WITH era_stats AS (
+        SELECT SUM(season_stats.hall_rating) AS rating,
+        player_id,
+        CASE WHEN year <= 1949 THEN 'early_baseball'
+          WHEN year > 1949 AND year <= 1969 THEN 'golden_days'
+          WHEN year > 1969 AND year <= 1987 THEN 'modern_baseball'
+          ELSE 'todays_game'
+          END AS era,
+
+        CASE WHEN year <= 1949 THEN 1
+          WHEN year > 1949 AND year <= 1969 THEN 2
+          WHEN year > 1969 AND year <= 1987 THEN 3
+          ELSE 4
+          END AS era_tiebreaker
+
+        FROM season_stats
+        JOIN players ON players.id = season_stats.player_id
+        WHERE players.eligibility = 'eligible'
```

Comment:
`WHERE players.hof = false` sounds reasonable. I'm pushing a change to add that because I also want to add some code to reset everyone's status to NULL before each time you run the task.

Adding that conditional means there are 1815 players who have a non-null `era_committee` value.

`rake era_committee:populate` is the command you want.